.github/workflows/release-nightly.yml

```diff
@@ -9,7 +9,12 @@ on:
   # Allows us to manually trigger a nightly
   # Since npm prevents duplicate releases we can run this at any time
   # As long as the commit hash has changed on main a release will be published
-  workflow_dispatch: {}
+  workflow_dispatch:
+    inputs:
+      wetRun:
```

Comment:
This already has to be triggered manually, is there a circumstance where we'd want to trigger this and not publish to npm? I can imagine debugging the actual script could require this, but then you could make a debug branch and add the flag there. If you want to keep it for future convenience so that we don't have to keep messing with the publish line, I would just call it `dryRun` and default to `false`

Reply:
Good point, I agree it is redundant and if we were testing we would be PR-ing a change anyway so we would be able to add the dry-run flag